app/lib/dttp/code_sets/countries.rb

```diff
@@ -0,0 +1,205 @@
+        "Bhutan" => { entity_id: "ad7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Bolivia" => { entity_id: "a57a4b74-ea28-e711-80c8-0050568902d3" },
+        "Bosnia and Herzegovina" => { entity_id: "8d7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Botswana" => { entity_id: "af7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Brazil" => { entity_id: "a97a4b74-ea28-e711-80c8-0050568902d3" },
+        "Brunei" => { entity_id: "a37a4b74-ea28-e711-80c8-0050568902d3" },
+        "Bulgaria" => { entity_id: "977a4b74-ea28-e711-80c8-0050568902d3" },
+        "Burkina Faso" => { entity_id: "957a4b74-ea28-e711-80c8-0050568902d3" },
+        "Burundi" => { entity_id: "9b7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Cambodia" => { entity_id: "537b4b74-ea28-e711-80c8-0050568902d3" },
+        "Cameroon" => { entity_id: "c77a4b74-ea28-e711-80c8-0050568902d3" },
+        "Canada" => { entity_id: "b57a4b74-ea28-e711-80c8-0050568902d3" },
+        "Cape Verde" => { entity_id: "d17a4b74-ea28-e711-80c8-0050568902d3" },
+        "Central African Republic" => { entity_id: "bb7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Chad" => { entity_id: "1b7c4b74-ea28-e711-80c8-0050568902d3" },
+        "Chile" => { entity_id: "c57a4b74-ea28-e711-80c8-0050568902d3" },
+        "China" => { entity_id: "c97a4b74-ea28-e711-80c8-0050568902d3" },
+        "Colombia" => { entity_id: "cb7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Comoros" => { entity_id: "577b4b74-ea28-e711-80c8-0050568902d3" },
+        "Congo (Democratic Republic)" => { entity_id: "b97a4b74-ea28-e711-80c8-0050568902d3" },
+        "Congo" => { entity_id: "bd7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Costa Rica" => { entity_id: "cd7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Croatia" => { entity_id: "2b7b4b74-ea28-e711-80c8-0050568902d3" },
+        "Cuba" => { entity_id: "cf7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Cyprus" => { entity_id: "537c4b74-ea28-e711-80c8-0050568902d3" },
+        "Czechia" => { entity_id: "d77a4b74-ea28-e711-80c8-0050568902d3" },
+        "Denmark" => { entity_id: "dd7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Djibouti" => { entity_id: "db7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Dominica" => { entity_id: "df7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Dominican Republic" => { entity_id: "e17a4b74-ea28-e711-80c8-0050568902d3" },
+        "East Timor" => { entity_id: "257c4b74-ea28-e711-80c8-0050568902d3" },
+        "Ecuador" => { entity_id: "e57a4b74-ea28-e711-80c8-0050568902d3" },
+        "Egypt" => { entity_id: "e97a4b74-ea28-e711-80c8-0050568902d3" },
+        "El Salvador" => { entity_id: "117c4b74-ea28-e711-80c8-0050568902d3" },
+        "Equatorial Guinea" => { entity_id: "197b4b74-ea28-e711-80c8-0050568902d3" },
+        "Eritrea" => { entity_id: "ed7a4b74-ea28-e711-80c8-0050568902d3" },
+        "Estonia" => { entity_id: "e77a4b74-ea28-e711-80c8-0050568902d3" },
+        "Eswatini" => { entity_id: "" },
```

Comment:
I'll be chasing Ben about this. Might have to take it out.

Reply:
We cant even map it to `Swaziland` as we'd planned as that isn't there either. Can we map it to 'South Africa' for now until Ben gets back to us. At least we'll have the correct data in Register if we leave the option there rather than taking it out.

Reply:
@gpeng Swaziland is available. I'm using it's entity ID for now.